src/plugins/discover/public/application/main/components/top_nav/open_alerts_popover.tsx

```diff
@@ -54,6 +60,13 @@ export function AlertsPopover({
    * Provides the default parameters used to initialize the new rule
    */
   const getParams = useCallback(() => {
+    if (isPlainRecord) {
+      return {
+        searchType: 'esqlQuery',
```

Comment:
I am seeing this UI for the first time and "Alerts generated" label looks confusing when you are just creating a new alert. Can we remove or rephrase it?

<img width="400" alt="Screenshot 2023-09-12 at 11 13 08" src="https://github.com/elastic/kibana/assets/1415710/88845cf2-39d9-4080-92e5-974b3833d9b3">

Reply:
This is not owned by us. If you have any recommendations for the alerting team it is better to create an issue for them